templates/openstack/what-is-openstack.html

```diff
@@ -465,325 +578,288 @@ <h2 class="u-sv3">OpenStack is big</h2>
+            <h3 class="p-heading--5">Whitepaper</h3>
+          </div>
+          <div class="grid-col-3">
+            <p>
+              <a href="/engage/cloud-pricing-report">Cloud pricing report 2022</a>
+            </p>
           </div>
         </div>
-
-        <div class="col-4 p-divider__block">
-          <div class="p-heading-icon--muted">
-            <div class="p-heading-icon__header">
-              {{ image(url="https://assets.ubuntu.com/v1/005c3416-webinar-white.svg",
-                            alt="",
-                            width="171",
-                            height="150",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-heading-icon__img"},) |
-              safe }}
-              <h4 class="p-heading-icon__title" style="color: #fff">Webinar</h4>
-            </div>
-            <h3 class="p-heading--4">
-              <a href="/engage/what-is-openstack" class="p-link--inverted">What is OpenStack and why does it matter?</a>
-            </h3>
+        <hr class="p-rule--muted" />
+        <div class="grid-row">
+          <div class="grid-col-1">
+            <h3 class="p-heading--5">Webinar</h3>
+          </div>
+          <div class="grid-col-3">
+            <p>
+              <a href="/engage/what-is-openstack">What is OpenStack and why does it matter?</a>
+            </p>
           </div>
         </div>
-        <div class="col-4 p-divider__block">
-          <div class="p-heading-icon--muted">
-            <div class="p-heading-icon__header">
-              {{ image(url="https://assets.ubuntu.com/v1/30037eac-datasheet-white.svg",
-                            alt="",
-                            width="171",
-                            height="150",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-heading-icon__img"},) |
-              safe }}
-              <h4 class="p-heading-icon__title" style="color: #fff">Datasheet</h4>
-            </div>
-            <h3 class="p-heading--4">
-              <a href="https://assets.ubuntu.com/v1/92fe24b8-Charmed.OpenStack_19.01.22.pdf"
-                 class="p-link--inverted">Canonical&rsquo;s Charmed OpenStack</a>
-            </h3>
+        <hr class="p-rule--muted" />
+        <div class="grid-row">
+          <div class="grid-col-1">
+            <h3 class="p-heading--5">Datasheet</h3>
+          </div>
+          <div class="grid-col-3">
+            <p>
+              <a href="https://assets.ubuntu.com/v1/92fe24b8-Charmed.OpenStack_19.01.22.pdf">Canonical&rsquo;s Charmed OpenStack</a>
+            </p>
           </div>
         </div>
       </div>
     </div>
   </section>
 
-  <section class="p-strip is-bordered">
-    <div class="u-fixed-width">
+  {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=true) -%}
```

Comment:
```suggestion
  {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=false) -%}
```
Similar suggestion here, this maintains the med layout that's already repeated on the majority of sections:
<img width="769" height="579" alt="image" src="https://github.com/user-attachments/assets/acdbc846-7bf5-402d-b364-7e72d2e474a8" />

vs
<img width="767" height="632" alt="image" src="https://github.com/user-attachments/assets/06089be2-387b-4ae8-8e71-4f2745d64076" />

Reply:
Thanks for the suggestion, @mtruj013 ! I agree with you here :) – I was also thinking this initially, but didn't realise we could do it for tiered lists so left it alone